source/core/NumericString.cpp

```diff
@@ -224,11 +224,14 @@ void DefaultFormatCode(Int32 count, StaticTypeAndData arguments[], TempStackCStr
         }
         switch (argType->BitEncoding()) {
 
-        case kEncoding_UInt:
-        case kEncoding_Enum: {
+        case kEncoding_UInt: {
             buffer->AppendCStr("%u");
         }
         break;
+            case kEncoding_Enum: {
```

Comment:
There are 60 files with whitespace problems, so I'm going to do this in a different PR.